jujugui/views.py

```diff
@@ -69,8 +69,13 @@ def config(request):
     settings = request.registry.settings
     request.response.content_type = 'application/javascript'
     sandbox_enabled = settings['jujugui.sandbox']
+    # If sandbox is enabled then set the password to "admin" so that the
+    # Juju GUI will automatically log in.
+    user, password = 'user-admin', 'admin'
+    if not sandbox_enabled:
+        user, password = settings['jujugui.user'], settings['jujugui.password']
     env_uuid = request.matchdict.get('uuid', 'sandbox')
-    baseUrl = settings.get('jujugui.baseUrl')
+    baseUrl = settings['jujugui.baseUrl']
```

Comment:
Because `jujugui.baseUrl` is always included in the dict. So using get was quite confusing.